src/Controls/samples/Controls.Sample/Pages/MainPage.cs

```diff
@@ -55,6 +55,16 @@ void SetupMauiLayout()
 			SemanticProperties.SetHeadingLevel((BindableObject)verticalStack.Children.Last(), SemanticHeadingLevel.Level1);
 			verticalStack.Add(new Label { Text = "This should be BOLD text!", FontAttributes = FontAttributes.Bold, HorizontalOptions = LayoutOptions.Center });
 			verticalStack.Add(new Label { Text = "This should be a CUSTOM font!", FontFamily = "Dokdo" });
+
+#if __ANDROID__
+			string fontFamily = "ionicons.ttf#";
+#elif WINDOWS
+			string fontFamily = "ionicons.ttf";
+#else
+			string fontFamily = "ionicons.ttf#";
+#endif
```

Comment:
If I'm not missing anything we can simplify this code

```suggestion
#if WINDOWS
			string fontFamily = "ionicons.ttf";
#else
			string fontFamily = "ionicons.ttf#";
#endif
```

Reply:
yea this isn't official code :-) Just silly code we need until something further upstream gets fixed and we can just do "ionicons"

Reply:
I see xD